source/ui.js

```diff
@@ -49,6 +49,24 @@ const printCommitLog = async repoUrl => {
 	};
 };
 
+const checkIgnoredFiles = async pkg => {
+	const ignoredFiles = await util.getNewAndUnpublishedFiles(pkg);
+	if (ignoredFiles === undefined || ignoredFiles.length === 0) {
+		return true;
+	}
+
+	console.log(`${chalk.bold('The following new files are not part of your published package:')}`);
```

Comment:
This should be a part of the `message` property of the prompt below (what I'd do is create a constant containing the whole message written clearly in a template string, then is it below).

Reply:
Moved the `message` property. Now it looks like:
**The following new files are not part of your published package:**
- foo/bar
**Continue?**